widgets/nuxeo-directory-checkbox.html

```diff
@@ -123,7 +123,9 @@
 
             _selected: {
               type: Array,
-              value: function() { return []; },
+              value() {
+                return [];
+              },
```

Comment:
```suggestion
              value: function() {
                return [];
              },
```

Reply:
@nelsonsilva i had that before but i got" eslint object-shorthand" that breaks the build

Reply:
nevermind, seems we're already using the shorthand in other places